### PR TITLE
update the windows pre-gyp script

### PR DESCRIPTION
**Pull request by fealebenpae**

### CODE REVIEW

File: scripts/build-node-pre-gyp.ps1 [hunk 1]
```diff
@@ -20,10 +20,10 @@ New-Item .\out -ItemType "directory"
 npm install --ignore-scripts
 
 foreach ($arch in "ia32", "x64") {
-    foreach ($version in "4.0.0", "5.0.0", "6.0.0", "7.0.0") {
+    foreach ($version in "4.0.0", "5.0.0", "6.0.0", "7.0.0", "8.0.0") {
         Remove-Item .\build, .\compiled -Recurse -Force -ErrorAction Ignore
         .\node_modules\node-pre-gyp\bin\node-pre-gyp.cmd rebuild --target_arch=$arch --target=$version
         .\node_modules\node-pre-gyp\bin\node-pre-gyp.cmd package --target_arch=$arch --target=$version
-        Copy-Item .\build\stage\node-pre-gyp\*.tar.gz -Destination .\out
+        Copy-Item .\build\stage\node-pre-gyp\**\*.tar.gz -Destination .\out
```
Comment (nnrealm) [inline scripts/build-node-pre-gyp.ps1]: two *s?

Comment (fealebenpae) [inline scripts/build-node-pre-gyp.ps1]: I suppose a single one could work, but I'm not that familiar with powershell globbing.

Comment (nnrealm) [inline scripts/build-node-pre-gyp.ps1]: Me neither! Just wondering if it was intentional.

Comment (fealebenpae) [inline scripts/build-node-pre-gyp.ps1]: Yeah, I want to glob all the tarballs in all the subfolders recursively.

**Pull request merged**